scripts/leodict.pl

```diff
@@ -380,6 +424,9 @@ sub sig_setup_changed {
 Irssi::settings_add_str($IRSSI{'name'}, 'leodict_default_options', '-en -both');
 Irssi::settings_add_int($IRSSI{'name'}, 'leodict_paste_max_translations', 2);
 Irssi::settings_add_bool($IRSSI{'name'}, 'leodict_paste_beautify', 1);
+Irssi::settings_add_str($IRSSI{'name'}, 'leodict_http_proxy_address', 'none');
```

Comment:
Instead of using `'none'` and forcing the user to possibly remember to use it to unset the proxy type, use `''` or `undef` and check for corresponding value in the `if statements` above